pkg/sql/catalog/tabledesc/validate.go

```diff
@@ -562,7 +562,7 @@ func validateMutation(m *descpb.DescriptorMutation) error {
 				"materialized view refresh mutation in state %s, direction %s", errors.Safe(m.State), errors.Safe(m.Direction))
 		}
 	case *descpb.DescriptorMutation_ModifyRowLevelTTL:
-		if m.Direction == descpb.DescriptorMutation_NONE {
+		if m.Direction != descpb.DescriptorMutation_NONE {
```

Comment:
Could this cause issues if a cluster is upgraded to a version containing this patch while there's a  TTL mutation in flight?

Reply:
I undid this after talking with @Xiang-Gu.